posts/2020-07-16-Rust-1.45.0.md

```diff
@@ -86,10 +86,10 @@ fn main() {
 ```
 
 On Rust 1.44.0, this happens to print "x: 0" on my machine. But it could
-print anything, or do anything: this is undefined behavior. But we have
-no `unsafe` code here. This is what we call a "soundness" bug, that is,
-it is a bug where the compiler does the wrong thing. We tag these bugs
-as
+print anything, or do anything: this is undefined behavior. But the `unsafe`
+keyword has not be specified in this code block. This is what we call a
+"soundness" bug, that is, it is a bug where the compiler does the wrong thing.
```

Comment:
```suggestion
keyword has not been specified in this code block. This is what we call a
"soundness" bug, where the compiler does the wrong thing.
```

minor suggestion re be vs been. using that is in the middle of a sentence is pretty weird as well, so maybe we can get two fixes in one.

Reply:
This suggestion is good re: grammar, but using "specified" here sounds odd, because specification has a different meaning, technically. 

I can appreciate this sentence maybe sounding odd, but I think moving it in this direction makes it sound *more* odd, not less.

Reply:
Oops, yeah the "be" was a typo.  Ironic.

Reply:
What about

But the `unsafe` keyword is nowhere to be found within this block of code.

?

Reply:
I prefer this latter formulation, fwiw.

Reply:
@josephtlyons yeah that sounds good to me!